docs/specification.md

```diff
@@ -1830,25 +1830,59 @@ Security is a paramount concern in A2A. Key considerations include:
 
 - **Transport Security:** Always use HTTPS with strong TLS configurations in production environments.
 - **Authentication:**
-    - Handled via standard HTTP mechanisms (e.g., `Authorization` header with Bearer tokens, API keys).
-    - Requirements are declared in the `AgentCard`.
-    - Credentials MUST be obtained out-of-band by the client.
-    - A2A Servers MUST authenticate every request.
+  - Handled via standard HTTP mechanisms (e.g., `Authorization` header with Bearer tokens, API keys).
```

Comment:
The mkdocs formatter seems to need four characters for doing nested lists, so these markdown changes are going to be problematic.